core/src/components/tabs/test/basic/index.html

```diff
@@ -30,8 +42,8 @@
           </ion-header>
           <ion-content class="ion-padding">
             <h1>Tab One</h1>
-            <ion-button expand="block" onclick="updateBadgeCount()">Update Badge Count</ion-button>
-            <ion-button color="secondary" expand="block" onclick="updateBadgeColor()">Update Badge Color</ion-button>
+            <button class="expand" onclick="updateBadgeCount()">Update Badge Count</button>
+            <button class="expand" color="secondary" onclick="updateBadgeColor()">Update Badge Color</button>
```

Comment:
& here for the `color` attribute.

Reply:
Removed in https://github.com/ionic-team/ionic-framework/pull/29422/commits/74d7dd7719335983a82364ba10fb54f9136522ea